From 78a6dcde1a2904c162450cf6a0b15c79d2f47176 Mon Sep 17 00:00:00 2001
From: Ivan Mironov <mironov.ivan@gmail.com>
Date: Thu, 18 Feb 2021 20:10:01 +0500
Subject: [PATCH 1/4] Enable debug info in release profile

---
 Cargo.toml | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Cargo.toml b/Cargo.toml
index 6e01beaeea2f..31c5845ec11c 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -85,3 +85,7 @@ resolver = "2"
 
 [profile.dev]
 split-debuginfo = "unpacked"
+
+[profile.release]
+opt-level = 3
+debug = true
-- 
2.33.0

