From 5c342a36c93d2ef6770e823ab54257ed7ab7fa82 Mon Sep 17 00:00:00 2001
From: Ivan Mironov <mironov.ivan@gmail.com>
Date: Thu, 18 Feb 2021 20:10:01 +0500
Subject: [PATCH 2/2] Enable LTO and debug info in release profile

---
 Cargo.toml | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Cargo.toml b/Cargo.toml
index 6c66b5046ceb..554d40834c85 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -79,3 +79,9 @@ exclude = [
 
 [profile.dev]
 split-debuginfo = "unpacked"
+
+[profile.release]
+opt-level = 3
+debug = true
+lto = "fat"
+codegen-units = 1
-- 
2.31.1

