From ae1c065c4a040222905d743cfe5f4366eeec24f3 Mon Sep 17 00:00:00 2001
From: Ivan Mironov <mironov.ivan@gmail.com>
Date: Thu, 18 Feb 2021 20:10:01 +0500
Subject: [PATCH 2/2] Enable LTO and debug info in release profile

---
 Cargo.toml | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Cargo.toml b/Cargo.toml
index 521859f764c8..57684808e641 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -80,3 +80,9 @@ exclude = [
 
 [profile.dev]
 split-debuginfo = "unpacked"
+
+[profile.release]
+opt-level = 3
+debug = true
+lto = "fat"
+codegen-units = 1
-- 
2.31.1

