From 7efb1e073535cc206ab57311d5695cc5ea186bc3 Mon Sep 17 00:00:00 2001
From: Ivan Mironov <mironov.ivan@gmail.com>
Date: Thu, 7 Oct 2021 07:57:32 +0500
Subject: [PATCH 2/3] Enable LTO

---
 Cargo.toml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Cargo.toml b/Cargo.toml
index 31c5845ec11c..09bc4e990441 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -89,3 +89,5 @@ split-debuginfo = "unpacked"
 [profile.release]
 opt-level = 3
 debug = true
+lto = "fat"
+codegen-units = 1
-- 
2.33.1

