From c019933cd9d6e48e152438b82d327ff4bd36adb2 Mon Sep 17 00:00:00 2001
From: Ivan Mironov <mironov.ivan@gmail.com>
Date: Thu, 18 Feb 2021 20:10:01 +0500
Subject: [PATCH 2/3] Enable LTO and debug info in release profile

---
 Cargo.toml | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Cargo.toml b/Cargo.toml
index 4d3e377cbbb9..d0992a8cfafd 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -83,3 +83,9 @@ resolver = "2"
 
 [profile.dev]
 split-debuginfo = "unpacked"
+
+[profile.release]
+opt-level = 3
+debug = true
+lto = "fat"
+codegen-units = 1
-- 
2.32.0

