From e67ee9e44193655a9e98c669e2740b2d145a76c4 Mon Sep 17 00:00:00 2001
From: Ivan Mironov <mironov.ivan@gmail.com>
Date: Thu, 7 Oct 2021 07:57:32 +0500
Subject: [PATCH 2/3] Enable LTO

---
 Cargo.toml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Cargo.toml b/Cargo.toml
index 573283576760..95e6bcd23eee 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -91,3 +91,5 @@ resolver = "2"
 [profile.release]
 opt-level = 3
 debug = true
+lto = "fat"
+codegen-units = 1
-- 
2.33.1

