From c98d56bfe3c8bbc6a9b7d8101b90e799cfc1c686 Mon Sep 17 00:00:00 2001
From: Ivan Mironov <mironov.ivan@gmail.com>
Date: Thu, 18 Feb 2021 20:10:01 +0500
Subject: [PATCH 2/3] Enable LTO and debug info in release profile

---
 Cargo.toml | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Cargo.toml b/Cargo.toml
index 0b24a45a2b40..547504190ed8 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -78,3 +78,9 @@ exclude = [
 
 [profile.dev]
 split-debuginfo = "unpacked"
+
+[profile.release]
+opt-level = 3
+debug = true
+lto = "fat"
+codegen-units = 1
-- 
2.31.1

