From e94238f5745853b64642281c2e0930ef0c2a4290 Mon Sep 17 00:00:00 2001
From: Ivan Mironov <mironov.ivan@gmail.com>
Date: Sun, 18 Jul 2021 11:29:24 +0500
Subject: [PATCH 3/3] Disable LTO

---
 Cargo.toml | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index d0992a8cfafd..eb9eb7115cd2 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -87,5 +87,3 @@ split-debuginfo = "unpacked"
 [profile.release]
 opt-level = 3
 debug = true
-lto = "fat"
-codegen-units = 1
-- 
2.32.0

