From fd4edac9e599b9b55334e3ae46307f8f42c5aff5 Mon Sep 17 00:00:00 2001
From: Ivan Mironov <mironov.ivan@gmail.com>
Date: Sun, 18 Jul 2021 11:29:24 +0500
Subject: [PATCH 3/3] Disable LTO

---
 Cargo.toml | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index 575c590f2c50..82580b8c545e 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -83,5 +83,3 @@ split-debuginfo = "unpacked"
 [profile.release]
 opt-level = 3
 debug = true
-lto = "fat"
-codegen-units = 1
-- 
2.31.1

