From 9b8e9d9a41f17e48437f61c340d7e2f693f03556 Mon Sep 17 00:00:00 2001
From: Ivan Mironov <mironov.ivan@gmail.com>
Date: Tue, 26 Oct 2021 12:03:04 +0500
Subject: [PATCH 4/4] Add watchtower option to add custom string into
 notifications

---
 watchtower/src/main.rs | 20 +++++++++++++++++---
 1 file changed, 17 insertions(+), 3 deletions(-)

diff --git a/watchtower/src/main.rs b/watchtower/src/main.rs
index 3dd06dd8aaa5..a8a7424e9c31 100644
--- a/watchtower/src/main.rs
+++ b/watchtower/src/main.rs
@@ -34,6 +34,7 @@ struct Config {
     monitor_active_stake: bool,
     unhealthy_threshold: usize,
     validator_identity_pubkeys: Vec<Pubkey>,
+    name_suffix: String,
 }
 
 fn get_config() -> Config {
@@ -134,6 +135,13 @@ fn get_config() -> Config {
                     no alerting should a Bad Gateway error be a side effect of \
                     the real problem")
         )
+        .arg(
+            Arg::with_name("name_suffix")
+                .long("name-suffix")
+                .value_name("SUFFIX")
+                .takes_value(true)
+                .help("Add this string into all notification messages after \"^solana-watchtower\"")
+        )
         .get_matches();
 
     let config = if let Some(config_file) = matches.value_of("config_file") {
@@ -159,6 +167,8 @@ fn get_config() -> Config {
     let monitor_active_stake = matches.is_present("monitor_active_stake");
     let ignore_http_bad_gateway = matches.is_present("ignore_http_bad_gateway");
 
+    let name_suffix = matches.value_of("name_suffix").unwrap_or("").to_owned();
+
     let config = Config {
         address_labels: config.address_labels,
         ignore_http_bad_gateway,
@@ -168,6 +178,7 @@ fn get_config() -> Config {
         monitor_active_stake,
         unhealthy_threshold,
         validator_identity_pubkeys,
+        name_suffix,
     };
 
     info!("RPC URL: {}", config.json_rpc_url);
@@ -337,8 +348,8 @@ fn main() -> Result<(), Box<dyn error::Error>> {
 
         if let Some((failure_test_name, failure_error_message)) = &failure {
             let notification_msg = format!(
-                "solana-watchtower: Error: {}: {}",
-                failure_test_name, failure_error_message
+                "solana-watchtower{}: Error: {}: {}",
+                config.name_suffix, failure_test_name, failure_error_message
             );
             num_consecutive_failures += 1;
             if num_consecutive_failures > config.unhealthy_threshold {
@@ -370,7 +381,10 @@ fn main() -> Result<(), Box<dyn error::Error>> {
                     humantime::format_duration(alarm_duration)
                 );
                 info!("{}", all_clear_msg);
-                notifier.send(&format!("solana-watchtower: {}", all_clear_msg));
+                notifier.send(&format!(
+                    "solana-watchtower{}: {}",
+                    config.name_suffix, all_clear_msg
+                ));
             }
             last_notification_msg = "".into();
             last_success = Instant::now();
-- 
2.33.0

